src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source-form.types.ts

```diff
@@ -84,6 +91,7 @@ export enum PreprocessKind {
 export enum EventTimeSourceKind {
     FROM_METADATA = "fromMetadata",
     FROM_PATH = "fromPath",
+    FROM_SYSTEM_TIME = "fromSystemTime",
 }
 
 export interface EditFormType {
```

Comment:
Quite a bad type name, very general

Reply:
Changed to` AddPollingSourceEditFormType`